com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 Version Updated
 The version number for this package has increased due to a version update of a related graphics package.
 
```

Comment:
Same here as in the other backport. You can remove those lines :)

Reply:
done.